docs/contributing/index.rst

```diff
@@ -1,7 +1,50 @@
 Contributing
 ============
 
+.. toctree::
+
 Contributing to RestrictedPython
+--------------------------------
+
+Legal requirements to contribute to RestrictedPython
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The projects under the zopefoundation GitHub organization, and therefore RestrictedPython, are open source and welcome contributions in different forms:
+
+* bug reports
+* code improvements and bug fixes
+* documentation improvements
+* pull request reviews
+
+For any changes in the repository besides trivial typo fixes you are required to sign the contributor agreement.
+See https://www.zope.dev/developer/becoming-a-committer.html for details.
+
+Please visit our `Developer Guidelines`_ if you'd like to contribute code changes and our `guidelines for reporting bugs`_ if you want to file a bug report.
+
+
+.. _`Developer Guidelines`: https://www.zope.dev/developer/guidelines.html
+.. _`guidelines for reporting bugs`: https://www.zope.dev/developer/reporting-bugs.html
+
+Preperations for Contributing
++++++++++++++++++++++++++++++
+
+If you want to contribute to this package, please prapare an development entvironment, that includes ``git``, ``tox`` and a posibility to have several Python Versions avaliable, for example ``pyenv``.
+Please read the part of `ref`Understanding How RestrictedPython works internally` first.
```

Comment:
Check that this reference actually exists. If not, then add a reStructuredText label just above the target section heading, such as:

```
.. my-label::

My Heading
----------
```

And refer to it:

```
:ref:`my-label`
```

It should then display as the heading just below the label.

```suggestion
Please read the part of :ref:`Understanding How RestrictedPython works internally` first.
```